shiny/ui/_navs.py

```diff
@@ -1200,7 +1200,7 @@ def navset_bar(
         provided, `fillable` makes the main content portion fillable.
     gap
         A CSS length unit defining the gap (i.e., spacing) between elements provided to
-        `*args`.
+        `*args`. This value is only used when the navbar is `fillable`.
```

Comment:
"navbar" strikes me as not quite the right word. That sounds like the bar portion of the navbar layout. 

I think we want `nav_panel()` is fillable? Which brings up an awkwardness that fillability of the `nav_panel()` is decided at the `navset_bar()` level, which can be `True` or a list of nav panel IDs that should be fillable.

Reply:
Agreed. I didn't know how to phrase it without creating confusion.